Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -741,6 +741,27 @@ def incident_to_incident_context(incident):
     return incident_context
 
 
+def fix_time_field(detection: dict, time_key: str):
+    """
+        Fix the value of the date to have only 6 figures after the ".".
+        The string representation of the created_timestamp value can contain from 6 to 9 figures after the dot,
+        for example: 2024-02-22T14:16:04.973070837Z. The template supports only 6 digits, so there is a need to remove the extra
+        digits to use datetime.strptime().
+
+        Args:
+            detection (dict): the detection.
+            time_key (str): the key of the wanted date&time field.
+    """
+    demisto.debug(f'fix_time_field {time_key=}')
+    str_date = detection[time_key]
+    split_date = str_date.split('.')
+    relevant_microseconds = split_date[1][:6]
+    # if 'Z' isn't in relevant_microseconds it means that it was removed since there was more than 5 digits in the microseconds.
+    fixed_date = f'{split_date[0]}.{relevant_microseconds}Z' if 'Z' not in relevant_microseconds else str_date
```

Comment:
If we get a timestamp with 6 digits (which according to the docstring is the minimal amount of digits) we will remove the Z as well. In which situation, we will have the Z after taking the first 6 chars?

Reply:
We will have the Z after taking the first 6 chars if there are 5 digits or less in in the microseconds.